hack/import_docs.sh

```diff
@@ -27,14 +30,18 @@ BRANCH_NAME=release-$release_version
 # Use the main branch if we are not targeting a stable version
 if ! [[ "${release_version}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
     BRANCH_NAME="main"
+    PREVIEW_RELEASE=1
```

Comment:
So, for the release candidate we're getting the documentation from `main`.
Two concerns:
- This script should not be re-run, say, a few days after, as the docs on main may have changed
- There are some placeholders in the documentation generation where we show the branch name. In this case we'll see `main`, not `1.24.0-rc1`. These mentions are not super-prominent, but they're there

Reply:
Ah, no, you already thought of that, I think....